src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int currentCapacity = DEFAULT_CAPACITY;
+    private int size;
+    private Node<K,V>[] buckets;
+
+    public MyHashMap() {
+        buckets = (Node<K,V>[]) new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size == currentCapacity * LOAD_FACTOR) {
```

Comment:
I think that you can make it like a field threshold and do like`if (size == threshold) {...`